Quorum/apis/git_api/git_manager.py

```diff
@@ -24,26 +24,36 @@ def __init__(self, customer: str) -> None:
         """
         self.customer = customer
         
-        self.customer_path = config.MAIN_PATH / self.customer / "modules"
-        self.customer_path.mkdir(parents=True, exist_ok=True)
+        self.customer_modules_path = config.MAIN_PATH / self.customer / "modules"
```

Comment:
We were discussing naming at the offsite... 
Maybe make it self.modules_path and review_module_path to make it shorter?

Reply:
Sure sounds good